charts/redis-ha/templates/redis-ha-statefulset.yaml

```diff
@@ -528,9 +528,10 @@ spec:
 {{- if .Values.extraContainers }}
 {{- toYaml .Values.extraContainers | nindent 6 }}
 {{- end -}}
-      {{- with .Values.priorityClassName | default .Values.global.priorityClassName }}
-      priorityClassName: {{ .Values.priorityClassName }}
-      {{- end }}
+      {{- with .Values.priorityClassName }}
```

Comment:
It would be better to modify the form as follows.
Also {{- end -}} seems to be missing.

```yaml
{{- with .Values.priorityClassName | default .Values.global.priorityClassName }}
priorityClassName: {{ . }}
{{- end }}
```

Reply:
@chikahisa-mikata Thank you :) 👍 to this statement.

Reply:
@DandyDeveloper 
I've issued a separate [PR](https://github.com/DandyDeveloper/charts/pull/295) for correction.